week1/task_学生成绩管理系统0.1.cpp

```diff
@@ -0,0 +1,62 @@
+#include <stdio.h> 
+int Not (float score[],int n);
+void Datescore(float score[],int n);
+int main()
+{
+	float score[40];
+	int n,i,a;
+	printf("please input the number of the students(n<=40):");
+	scanf("%d",&n);
+	printf("please input the score of the students:\n");
+	for (i=0;i<n;i++)
+	{
+		scanf("%f",&score[i]);
+	}
+	a=Not (score,n);
+	printf("²»¼°¸ñµÄÈËÊýÎª£º%d\n",a);
+	Datescore(score,n);
+	for (i=0;i<n;i++)
+	{
+		printf("%.1f\n",score[i]);
+	}
+	return 0;
+} 
+int Not (float score[],int n)
+{
+	int i,k=0;
+	for (i=0;i<n;i++)
+	{
+		if (score[i]<60)
+		{
+			k+=1;
+		}
+	}
+	return k;
+}
+void Datescore(float score[],int n)
+{
+	int j,i,k,temp;
+	for (i=0;i<n-1;i++)

```

Comment:
for variables used in iteration, how about using inline declaration. For example,

```
for (int i = 0; i < n - 1; i++)
```

For high level programming language, we are always doing this anyway:)